4주차/[Lv.3]_등굣길/주다현.js

```diff
@@ -0,0 +1,17 @@
+//런타임 에러가 나는 코드랍니다... 아직 해결하지 못했어요ㅠㅠㅠㅠ
+
+function solution(m, n, puddles) {
+  const arr = Array.from(Array(n), () => Array(m).fill(1));
+  puddles.forEach((v) => {
+    const [x, y] = v;
+    arr[x - 1][y - 1] = 0;
```

Comment:
```suggestion
    arr[x - 1][y - 1] = null;
```
개인적인 취향이지만 저의 경우 `물이 잠긴 지역`을 구분하기 위해 `null`로 해당하는 배열 안의 값을 선언하겠습니다.